documentation/prover-incentives.md

```diff
@@ -0,0 +1,72 @@
+# Efficient prover incentives
+
+This document describes an ahead of time auction mechanism that allows provers to bid for the right to prove multiple publications for a rollup. The mechanism is guided with the following principles in mind:
+
+- Decouple the roles of proposing and proving.
+- Assigning proving rights to one prover at a time to avoid redundant efforts, which results in cheaper costs.
+- Providing proposers with predictable costs when submitting publications.
+- Simple design and minimal L1 gas costs.
+
+*The specifics of the mechanism are still being discussed, and some of the details might change as we do more research on the incentives.*
+
+## The problem
+
+Verifying proofs on Ethereum L1 is a [gas-intensive operation](https://docs.alignedlayer.com/#why-are-we-building-aligned). For example, Groth16 proofs, among the cheapest, cost around 250k gas, while STARKS can exceed 1M gas. As a result, most zk-rollups call their inbox contract's verification function only sporadically.
+
+For based validity rollups, achieving a similar property raises the question: Who should verify batches submitted to L1, and how can proposers be incentivized to submit valid batches if proofs aren’t generated immediately? A common solution is to require proposers to deposit a bond when submitting a batch. If they fail to provide a validity proof within a set time window, they lose their stake, and proving becomes permissionless. This approach is currently used by [Taiko].
```

Comment:
"[Taiko]". Is this supposed to be a link?

Reply:
Yes, [this](https://docs.taiko.xyz/resources/terminology/#validity-bond) should be it